#! /bin/sh -e

# DP: Remove -u option from vboxplay.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- isdnutils-3.3.orig/vbox/utils/vboxplay.in
+++ isdnutils-3.3/vbox/utils/vboxplay.in
@@ -44,7 +44,9 @@
 		# if anyone has a better suggestion,
 		# send email to isdnutils@packages.debian.org!
                 # thankfully, aplay understands '-' to mean stdin
-		DOPLAY="$PLAY -q -u -"
+                # 20021115 removed -u option due to #165757,
+                # aplay doesn't understand that option (anymore?)
+		DOPLAY="$PLAY -q -"
 	fi
 fi
 if [ ! -x $PLAY ]; then	# use play (from sox) instead
